mux/src/tab.rs

```diff
@@ -1038,7 +1039,11 @@ impl Tab {
     /// edge intersection.
     pub fn activate_pane_direction(&self, direction: PaneDirection) {
         if self.zoomed.borrow().is_some() {
-            return;
+            if configuration().unzoom_on_switch_pane {
+                self.toggle_zoom();
+            } else {
+                return;
+            }
```

Comment:
I prefer early-return style:

```suggestion
            if !configuration().unzoom_on_switch_pane {
                return;
            }
            self.toggle_zoom();
```